custom_liferay_overwrite.css

```diff
@@ -1,38 +1,59 @@
 /* Makes Navigation bar go over entire page width 
 and Removes Side margins from Liferay Homepage */
+
 #wrapper {
   padding: 0;
   margin: inherit;
 }
 
 /* Moves logo and site number to the right */
-#banner > #heading {
+
+#banner>#heading {
   padding: 0px 5px 0px 5px;
 }
 
 /* logo sizing */
+
 #heading .custom-logo {
   height: 5rem;
 }
 
-#heading .logo.custom-logo > img{
+#heading .logo.custom-logo>img {
   width: inherit;
   height: inherit;
   object-fit: contain;
 }
 
+/* Changes background color and font-size for navbar */
+
+#navigation .navbar-inner {
+  font-size: 1.3em;
+  background-color: var(ncct-dark) !important;
```

Comment:
Do you need `!important` here? We should avoid it wherever possible.

Reply:
I tried it without !important but then it resolves back to the default liferay styling for the navigation bar

Reply:
```suggestion
  background: var(--ncct-dark);
```

Reply:
This should work. I personally would leave the greyish background as is though.